src/markdown-pages/docs/running-simbank-tests/web-app-integration-test.md

```diff
@@ -0,0 +1,162 @@
+
+You can <a href="https://github.com/mozilla/geckodriver/releases" target="_blank"> download Gecko driver from GitHub</a>. 
+
+You must define the default and local driver properties in the CPS, as shown in the following example:
+
+```
+selenium.default.driver=FIREFOX
+selenium.local.driver.FIREFOX.path=<path/to/geckodriver>
+```
+
+### Using the Docker Manager 
+
+The Selenium Manager has a dependency on the Docker Manager in order to run. Some set up is required for the Docker Manager in the CPS properties file. To configure the Docker Manager, set the following CPS properties:  
+
+```
+docker.dse.engine.PRIMARY=LOCAL
+docker.default.engines=LOCAL
+docker.engine.LOCAL.hostname=127.0.0.1
+docker.engine.local.port=2375
+docker.engine.local.max.slots=10
+docker.container.TAG.name=simbank-webapp
+```
+
+These properties allow local test runs to access the local Docker Engine when the TCP port of the local Docker Engine is enabled.
+
+
+After updating the CPS properties, run the following terminal commands to open a TCP socket for accessing the Dokcer API:
+
+```
+docker pull alpine/socat
+docker run -d -p 127.0.0.1:2376:2375 -v /var/run/docker.sock:/var/run/docker.sock alpine/socat tcp-listen:2375,fork,reuseaddr unix-connect:/var/run/docker.sock
+```
+
+Test that the container is working correctly by running the following command:
+
+```
+mvn install
+docker image build -t simbank-webapp .
+docker run -p 8080:8080 -d simbank-webapp
+```
+
+### Troubleshooting
+
+If the container is not working correctly, for example, compilation and runtime errors are returned, check the version of tomcat in the Dockerfile. You might need to edit the tomcat version to a version that is compatible with Java 11, for example, ```FROM tomcat:8.5.82-jre11-temurin```.
+
+
+### About the WebAppIntetgrationTest
+
+To run the test, follow the same steps as for `SimBankIVT.java` but using the test class name `WebAppIntetgrationTest` instead of `SimBankIVT`. Don't forget that you need to launch [Galasa SimBank](/docs/getting-started/simbank) before running the test.
+
+This test performs a similar function to the `ProvisionedAccountCreditTest.java` but includes the use of the Selenium Manager. The Selenium Manager enables the test to run Selenium WebDrivers which drive the SimBank Web Application that is provided with Galasa SimBank. 
+
+The `WebAppIntetgrationTest` generates a unique account number to use in the test and provisions the account with a randomly generated opening balance by using a 3270 emulator that connects to the web application which opens in Firefox. A batch job opens the account and fills in a form to credit the account with the opening balance. The 3270 emulator then connects to the web application, searches for the account number and retrieves the balance.  
```

Comment:
I think there is confusion about what the test does.  Let me explain:
1. A Random Account number is generated
2. the 3270 application is used to ensure that this account number does not exist
3. a batch job is run to create the account
4. selenium drives the web application to credit the account with some funds
5. the 3270 application is run again to validate the funds were added

Reply:
thank you, have amended accordingly